### PR TITLE
LPD-20788 - use correct companyID when importing an organization

### PR DESCRIPTION
**Pull request by wanderlast**
Hi @smottal,

https://liferay.atlassian.net/browse/LPD-20788, tied to LPP.

This builds off the work done in https://liferay.atlassian.net/browse/LPD-19194, but makes the adjustment of grabbing the companyID based on CompanyThreadLocal instead of based on the country itself. This prevents potential issues importing in LXC where we may end up accidentally targeting the wrong database partition.

### CODE REVIEW

**Comment by wanderlast** [comment]
ci:test:sf

**Comment by wanderlast** [comment]
ci:test:um

**Comment by wanderlast** [comment]
ci:test:relevant

File: modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/exportimport/data/handler/CountryStagedModelDataHandler.java [hunk 1]
```diff
@@ -85,7 +86,7 @@ protected void doImportStagedModel(
 		throws Exception {
 
 		Country existingCountry = _countryLocalService.fetchCountryByA2(
-			country.getCompanyId(), country.getA2());
+			CompanyThreadLocal.getCompanyId(), country.getA2());
```
Comment (smottal) [inline modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/exportimport/data/handler/CountryStagedModelDataHandler.java:89]: @wanderlast is it possible to get the companyId from the portletDataContext?

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by wanderlast** [comment]
ci:test:um

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**